src/locale/locales/fr/messages.po

```diff
@@ -6623,7 +6624,7 @@ msgstr "Spam"
 
 #: src/lib/moderation/useReportOptions.ts:55
 msgid "Spam; excessive mentions or replies"
-msgstr "Spam ; mentions ou réponses excessives"
+msgstr "Spam ; mentions ou réponses excessives"
```

Comment:
```suggestion
msgstr "Spam ; mentions ou réponses excessives"
```
A non-breaking space got accidentally removed here (I think the only string where a semicolon is used?)